src/common/tracer.h

```diff
@@ -0,0 +1,31 @@
+#ifndef TRACER_H_
+#define TRACER_H_
+
+#define SIGNED_RIGHT_SHIFT_IS 1
+#define ARITHMETIC_RIGHT_SHIFT 1
+
+#include <arpa/inet.h>
```

Comment:
you don't need this include

Reply:
I need otherwise it wont compile

Reply:
maybe you need that in the cc file, but not here